.github/workflows/publish-release.yml

```diff
@@ -86,3 +86,26 @@ jobs:
         with:
           npm-token: ${{ secrets.NPM_TOKEN }}
           npm-tag: ${{ steps.get-release-tag.outputs.tag }}
+
+  get-release-version:
+    runs-on: ubuntu-latest
+    needs: npm-publish
```

Comment:
We might need some environment changes as well, depending on whether we want me to be able to publish the iframe-execution-environment.

I can't do that in the current setup, but this would give me that permission unless we make changes. @ritave

Reply:
What do you need me to do?

Reply:
We need to discuss whether I should be able to publish everything or not 😅 

If not, we need to make some permissions changes on this repo I think 🤔

Reply:
Oh actually, since it is separate environments then we might be able to keep the permission as-is?